src/_articles/archive/zones.md

```diff
@@ -193,20 +198,25 @@ they still execute.
 As a consequence a zoned error handler might
 be invoked multiple times.
 
-Also note that an _error zone_—a zone that has an error handler—might
-handle errors that originate in a child (or other descendant)
-of that zone.
+Any zone that includes a zoned error handler is an _error zone_.
+An error zone might handle errors that originate
```

Comment:
**question:** Can you not use a less complex word? Would `start` or `begin`?

Reply:
Skipping, "originate" captures the nuance, "start" or "begin" implies some error lifecycle which isn't really the case. Also "originate" is not so complex that it's jargon or derailing.

Reply:
OK. Non-blocking, so makes sense.